miqi/skills/qraft-workflowspec-export/scripts/upload_run.py

```
@@ -0,0 +1,205 @@
+    if 400 <= status < 500:
+        message = ""
+        try:
+            data = resp.json()
+            message = str(data.get("message") or data.get("msg") or "")
+        except ValueError:
+            message = body_text[:200]
+        return "BAD_REQUEST", f"上传失败（HTTP {status}）：{message or '请求参数错误'}"
+    if status >= 500:
+        return "SERVER_ERROR", f"上传失败（HTTP {status}）：Qraft 服务端异常，请稍后重试"
+    if status == 200:
+        # 实测成功时 body 为纯文本 ok；若返回 JSON 业务信封且 code != 200，
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `；` (FULLWIDTH SEMICOLON). Did you mean `;` (SEMICOLON)?

(RUF003)